war/src/main/webapp/WEB-INF/elasticsearch.yml

```diff
@@ -31,39 +33,20 @@ path.home: es
 #
 path.data: ${exo.data.dir}
 
-# Path to where plugins are installed:
-#
-path.plugins: es/plugins
```

Comment:
It was used to install new plugins (by customers for example). What is the new path ? Is it still possible ?